clang/lib/CodeGen/CodeGenFunction.cpp

```diff
@@ -2909,18 +2909,16 @@ static int getPriorityFromAttrString(StringRef AttrStr) {
   AttrStr.split(Attrs, ';');
```

Comment:
Should the function return `unsigned` instead of `int` now?

Reply:
Sure.